bin/update-helper.sh

```diff
@@ -120,7 +120,7 @@ function consolidate_configuration() {
 ## Defaults:
 author_commit=""
 drush="vendor/bin/drush"
-updates="composer show --locked --outdated --name-only"
+updates="composer show --locked --direct --name-only"
```

Comment:
By doing this we would try to update every direct dependence even there aren't updates. What is the reason to do that?

My only concern is that if there aren't needed updates or just a few, the script will be really slow to provide feedback.

Reply:
Ok i've read the full issue. Solving.